Source/OrchBase/Composer.cpp

```diff
@@ -1635,8 +1641,11 @@ void Composer::formDevTOutputPinHandlers(devTypStrings_t* dTypStrs)
         if ((*pinO)->pMsg->pPropsD)
         {
             handlers_cpp << "   pkt_" << (*pinO)->pMsg->Name();
-            handlers_cpp << "_pyld_t* message = static_cast<pkt_";
+            handlers_cpp << "_pyld_t* message = ";
```

Comment:
I have just noticed (and found in some testing) that this is actually broken. The "=" at the end of 1644 should NOT be there